src/components/Editor/DebugLine.js

```diff
@@ -66,29 +54,24 @@ export class DebugLine extends Component<Props, State> {
 
     const { line, column } = toEditorPosition(sourceId, selectedFrame.location);
 
```

Comment:
we should only scroll the editor from the Editor component. And, anyways this should be redundant.